stable/kong/README.md

```diff
@@ -79,15 +79,12 @@ document.
 ## Kong Enterprise
 
 If using Kong Enterprise, several additional steps are necessary before
-installing the chart. At minimum, you must:
-* Create a [license secret](#license).
-* Set `enterprise.enabled: true` in values.yaml.
-* Update values.yaml to use a Kong Enterprise image. If needed, follow the
-instructions in values.yaml to add a registry pull secret.
+installing the chart.
 
-Reading through
-[the full list of Enterprise considerations](#kong-enterprise-parameters)
-is recommended.
+Please read through
+[Kong Enterprise considerations](#kong-enterprise-parameters)
+to understand the prerequisites and various other settings that are enterprise
+specific.
```

Comment:
Any reason to remove the quick instructions here? These should remain the bare minimum to set up an Enterprise install. The original intent was providing a quick set of steps to get a basic environment up and running, as the complete Enterprise section is fairly long.

Reply:
The reason for this is to move all enterprise specific stuff to the section below, in one place.
Also, folks set up license but forget to set the docker registry creds, hence, I move those intor prerequisites.
Let me re-frame the below test to highlight that the minimum you need is those two secrets and you should be able to get going.

Does that work?

Reply:
Perhaps move the existing text under the *Install* section? I know it's duplicating some of the content in the later section, but it's not anything I'd expect to change, and shouldn't require much (if any) revision anytime soon. I prefer calling this out early if possible.

If not, I'd recommend replacing the existing [second Enterprise Overview paragraph](https://github.com/helm/charts/blame/9474ab5f3cd2ff172547ca0d22679fc587c7ed68/stable/kong/README.md#L333-L336) with the bulleted list; it makes it clearer what's required than the existing paragraph format. The third point should probably link to the larger registry section also:

```
Update values.yaml to use a Kong Enterprise image and [configure your registry credentials](#kong-enterprise-docker-registry-access)
```

Reply:
I tried the first approach, can you take a look?